drivers/usb/device/usb_dc_stm32.c

```diff
@@ -987,20 +987,16 @@ void HAL_PCD_DataInStageCallback(PCD_HandleTypeDef *hpcd, u8_t epnum)
 void HAL_PCDEx_SetConnectionState(PCD_HandleTypeDef *hpcd, uint8_t state)
 {
 	struct device *usb_disconnect;
+	u8_t gpio_flags = DT_INST_0_ST_STM32_USB_DISCONNECT_GPIOS_FLAGS;
```

Comment:
This really should be `gpio_devicetree_flags_t`.  Or, better, `u32_t` then use the `state` parameter to add in either `GPIO_OUTPUT_ACTIVE` or `GPIO_OUTPUT_INACTIVE`.  Or better just do something like this
```
gpio_pin_configure(usb_disconnect,
		   DT_INST_0_ST_STM32_USB_DISCONNECT_GPIOS_PIN,
		   DT_INST_0_ST_STM32_USB_DISCONNECT_GPIOS_FLAGS
                   | (state ? GPIO_OUTPUT_ACTIVE : GPIO_OUTPUT_INACTIVE));
```

Reply:
Done